src/infra/sequelize/queries/frise/getPtfDTO.ts

```diff
@@ -0,0 +1,45 @@
+import { User } from '@entities'
+import { PtfDTO, ProjectStatus } from '@modules/frise'
+import { makeDocumentUrl } from '../../../../views/components/timeline/helpers/makeDocumentUrl'
+
+type PtfDonnéesPourDTO =
+  | {
+      ptfDateDeSignature: Date
+      ptfFichier: { filename: string; id: string }
+    }
+  | {
+      ptfDateDeSignature: null
+      ptfFichier: null
+    }
+  | null
+
+export const getPtfDTO = ({
+  ptf,
+  user,
+  projetStatus,
+}: {
+  ptf: PtfDonnéesPourDTO
+  user: User
+  projetStatus: ProjectStatus
+}): PtfDTO | undefined => {
+  if (!ptf) return
+  if (projetStatus !== 'Classé') return
```

Comment:
vu la simplificité des vérifs on peut faire du oneline ici jpense

```suggestion
  if (!ptf || projetStatus !== 'Classé') return
```